File: exam-system-java/server/src/main/java/com/yyjy/exam/question/repository/QuestionChoicesRepository.java
package com.yyjy.exam.question.repository;

import com.yyjy.exam.entity.question.entity.QuestionChoices;
import org.babyfish.jimmer.spring.repository.JRepository;

public interface QuestionChoicesRepository extends JRepository<QuestionChoices, Long> {

    void deleteByQuestionId(long questionId);
}

File: exam-system-server/server/src/main/kotlin/com/yyjy/repository/QuestionChoicesRepository.kt
package com.yyjy.repository

import com.yyjy.models.entity.QuestionChoices
import org.babyfish.jimmer.spring.repository.KRepository

/**
 * @author Nyxcirea
 * date 2026/4/1
 * description: TODO
 */
interface QuestionChoicesRepository : KRepository<QuestionChoices, Long> {
    fun deleteByQuestionId(questionId: Long)
}
